src/modules/0.0.22/aws_rds/entity/parameter_group.ts

```diff
@@ -58,9 +58,14 @@ export enum ParameterGroupFamily {
 }
 
 @Entity()
+@Unique('paragrp_name_region', ['name', 'region'])
+@Unique('paragrp_id_region', ['id', 'region'])
 export class ParameterGroup {
-  @PrimaryColumn()
+  @PrimaryGeneratedColumn()
+  id?: number;
```

Comment:
`@PrimaryGeneratedColumn` should always have a value, shouldn't it? But it's nullable here. And it's having a `@ Unique` constraint together with `region` which is strange since `id` should always be unique, and combining it with anything would make that pair unique as well.

Reply:
So two things:

1. We mark it as nullable here despite it always coming back with a value from TypeORM because we construct these objects with our mapper code and they won't have database IDs, which is why that's that way.
2. Both of those `@Unique` decorators being there must be some sort of accidental rebase glitch? I originally did `['id', 'region']` then remembered it should have been `['name', 'region']` and changed it, and I am just as surprised as you that `['id', 'region']` somehow got back into the codebase.

Reply:
Wait, no. I take that back, but I will add a comment on this. There's a really weird Postgres unique constraint error if I don't do this: https://github.com/iasql/iasql-engine/actions/runs/3216578406/jobs/5258602677#step:5:7128 because of how the RDS entity joins on the ParameterGroup entry.

Reply:
Got it. So if you do the join by those two columns, you have to explicitly define a unique constraint. Somehow weird but in some sense it also makes sense :D .